docs/prompting/zero_shot/role_prompting.md

```diff
@@ -1,7 +1,65 @@
 ---
-title: ""
+title: "Role Prompting"
 description: ""
-keywords: ""
+keywords: "prompt, prompting, prompt engineering, llm, gpt, openai, role prompting, persona prompting, model, AI, python, instructor, zero-shot"
 ---
 
-[wip]
+Role prompting<sup><a href="https://arxiv.org/abs/2310.00746">1</a>, <a href="https://arxiv.org/abs/2311.10054">2</a></sup>, or persona prompting<sup><a href="#ref-3">3</a>, <a href="https://arxiv.org/abs/2307.05300">4</a></sup>, assigns a specific role to the model in the prompt ("You are ..."). This can improve outputs for open-ended tasks<sup><a href="https://dl.acm.org/doi/10.1145/3411763.3451760">5</a></sup> and improve accuracy on benchmarks<sup><a href="https://arxiv.org/abs/2311.10054">2</a></sup>.<sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>
```

Comment:
Add a header here # Role Prompting so that the TOC is populated

I suggest doing a template form

> You are a {role name}. You need to {describe task here}. {quantifiers if possible}

then using the example below. It might be useful to then show how we can do so for a specific task. I like the labelling task.

You are an expert AI labelling system.... etc